Zero Bounce iOS SDKTests/Zero_Bounce_iOS_SDKTests.swift

```diff
@@ -7,28 +7,572 @@
+        let mock = Mock(url: apiEndpoint, dataType: .json, statusCode: 200, data: [.get: mockedData])
+        mock.register()
 
+        let requestExpectation = expectation(description: "Request should finish")
+        ZeroBounceSDK.shared.deleteFile(fileId: fileId) { result in
+            defer { requestExpectation.fulfill() }
+            switch result {
+            case .Success(let response):
+                NSLog("validate success response=\(response)")
+                XCTAssertEqual(response.success, true)
+            case .Failure(let error):
+                NSLog("validate failure error=\(String(describing: error))")
+                XCTAssertTrue(false)
+            }
+        }
+
+        wait(for: [requestExpectation], timeout: 60)
+    }
+    
+    func testScoringDeleteFile() {
+        
+        let fileId = "file_id"
+        let apiEndpoint = URL(
+            string: "\(ZeroBounceSDK.shared.bulkApiBaseUrl)/scoring/deletefile?api_key=\(ZeroBounceSDK.shared.apiKey!)&file_id=\(fileId)"
+        )!
+        
+        let fileDeleteResponse = ZBDeleteFileResponse(success: true, message: nil, fileName: fileId, fileId: fileId)
+        
+        let mockedData = try! JSONEncoder().encode(fileDeleteResponse)
+        let mock = Mock(url: apiEndpoint, dataType: .json, statusCode: 200, data: [.get: mockedData])
+        mock.register()
+
+        let requestExpectation = expectation(description: "Request should finish")
+        ZeroBounceSDK.shared.scoringDeleteFile(fileId: fileId) { result in
+            defer { requestExpectation.fulfill() }
+            switch result {
+            case .Success(let response):
+                NSLog("validate success response=\(response)")
+                XCTAssertEqual(response.success, true)
+            case .Failure(let error):
+                NSLog("validate failure error=\(String(describing: error))")
+                XCTAssertTrue(false)
+            }
+        }
+
+        wait(for: [requestExpectation], timeout: 60)
+    }
+    
+    func testGetFile() {
+        
+        let fileId = "fileId"
+        let apiEndpoint = URL(
+            string: "\(ZeroBounceSDK.shared.bulkApiBaseUrl)/getFile?api_key=\(ZeroBounceSDK.shared.apiKey!)&file_id=\(fileId)"
+        )!
+        
+        let getFileResponse = ZBGetFileResponse(localFilePath: "path")
+        
+        let mockedData = try! JSONEncoder().encode(getFileResponse)
+        let mock = Mock(url: apiEndpoint, dataType: .json, statusCode: 200, data: [.get: mockedData])
+        mock.register()
+
+        let requestExpectation = expectation(description: "Request should finish")
+        ZeroBounceSDK.shared.getFile(fileId: fileId) { result in
+            defer { requestExpectation.fulfill() }
+            switch result {
+            case .Success(let response):
+                NSLog("validate success response=\(response)")
+                XCTAssertTrue(response.localFilePath != nil)
```

Comment:
why don't we check that localFilePath == "path"?

Reply:
the path depends on the simulator